modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/extension/creole_dialog_definition.js

```diff
@@ -47,7 +47,16 @@ CKEDITOR.on('dialogDefinition', (event) => {
 			infoTab.remove('cmbWidthType');
 			infoTab.remove('cmbWidthType');
 			infoTab.remove('htmlHeightType');
-			infoTab.remove('txtBorder');
+
+			var loadedPlugins = CKEDITOR.plugins.loaded;
```

Comment:
Why not simply `if (infoTab.get('txtBorder')`??

Reply:
I thought that checking if the `creole` and `showborders` plugin weren't loaded should also be taken into account - 
I'll edit that